controllers/consoleplugin/consoleplugin_test.go

```diff
@@ -110,13 +110,14 @@ func TestContainerUpdateCheck(t *testing.T) {
 	//equals specs
 	plugin := getPluginConfig()
 	loki := &flowsv1alpha1.FlowCollectorLoki{URL: "http://loki:3100/", TenantID: "netobserv"}
-	builder := newBuilder(testNamespace, testImage, &plugin, loki, &certWatcher)
+	spec := &flowsv1alpha1.FlowCollectorSpec{ConsolePlugin: plugin, Loki: *loki}
+	builder := newBuilder(testNamespace, testImage, spec, &certWatcher)
```

Comment:
(Minor) For `loki`, there's no point in taking the address of (&) only to dereference it back (\*) in the next line.  Also, you might then want `spec` to be consistent.

```go
loki := flowsv1alpha1.FlowCollectorLoki{URL: "http://loki:3100/", TenantID: "netobserv"}
spec := flowsv1alpha1.FlowCollectorSpec{ConsolePlugin: plugin, Loki: loki}
builder := newBuilder(testNamespace, testImage, &spec, &certWatcher)
```

If you do this, make sure you change it in all the other places in the code.

Reply:
Fixed, thanks!